internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -725,6 +725,38 @@ func (s *LoadBalancedWebService) convertGracePeriod() *int64 {
 	return aws.Int64(int64(manifest.DefaultHealthCheckGracePeriod))
 }
 
+func (s *LoadBalancedWebService) convertImportedALB() (*template.ImportedALB, error) {
+	if s.importedALB == nil {
+		return nil, nil
+	}
+	var hasMultipleListeners bool
+	if len(s.importedALB.Listeners) == 2 {
```

Comment:
It seems so far that transformers do not know that we accept either 1 or 2 listeners - which is good, because if we relax this constraint in the future, we would only need to change the code in `validateALBRuntime` in `cli/deploy/lbws.go`, and the CloudFormation template. We wouldn't need to remember to change the logic here. My suggesetion is below:
```suggestion
	if len(s.importedALB.Listeners) != 1 {
```

Reply:
Ah yes, that was my reasoning behind leaving it as "multiple" and not changing it to "two" after we decided to error out if not two. But I hadn't thought to change this-- good idea.